style.css

```diff
@@ -0,0 +1,143 @@
+*{
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+body{
+  background: #fff;
+}
+
+.header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: #D8DBDE;
+}
+
+.search-bar{
+  flex-grow: 1;
+  display: flex;
+  margin: 0 2rem 0 4rem;
+  
+}
+
+.search-input{
+  height: 40px;
+  width: 100%;
+  font-size: 17px;
+  padding: .4em;
+  margin-left: 150px;
+  border: 1px solid #777;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.search-btn {
+  margin-right: 300px;
+  display: flex;
+  align-items: center;
+  padding: 0 1.25rem;
+  border: 1px solid #777;
+  background-color: #f8f8f8;
+  border-left: none;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  cursor: pointer;
+  height: auto;
+}
+
+.search{
+  background-color: #f0f0f0;
+  height: 38px;
+  width: 30px;
+}
+
+.mic{
+  margin-left: -310px;
+  height: 34px;
+}
```

Comment:
Your use of the class property of CSS is very impressive. And it would be a lot better if you use float and position property to position your mic element.

Reply:
[Optional] What do you think about giving a detailed explanation as to why change is requested 🤔 

Instead of _😎I suggest you use float and position properties on the margin👍🏼_ , we can give a more detailed reason such as _😎I recommend you use float and position properties on the margin in order to have a nice and structured positioning and match the required wireframe provided👍🏼_